elser/operations/bm25-queries.json

```diff
@@ -0,0 +1,120 @@
+{
```

Comment:
I wonder if we can use the 30k queries instead of a subset? We can generate the query for the queries file directly (randomly or not). Then we'll be able to report the recall using the annotations from the test dataset?

Reply:
I've added a Python parameter source class to produce the queries from a JSON file containing the ELSER tokens. The file only contains 1 set of tokens for now (so it is always the same query) but it will be trivially easy to add more in future.